### PR TITLE
Acrs 137 dob validation child

### PR DESCRIPTION
**Pull request by michaelhartddat**
## What? 

[ACRS-137 dob validation /child-details](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-137)

Added the `isUnder18` validation to the /child-details date-of-birth field. dob is configured from `dobCutoff`

### CODE REVIEW

File: apps/acrs/fields/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const dateComponent = require('hof').components.date;
 const after1900Validator = { type: 'after', arguments: ['1900'] };
 const countries = require('hof').utils.countries().concat([{ value: 'Unknown', label: 'Unknown' }]);
 const isInCountriesList = value => countries.some(country => country.value === value);
+const isUnder18 = value => ! require('../../../lib/utilities').isOver18(value);
```
Comment (jamiecarterHO) [inline apps/acrs/fields/index.js:8]: tidy

File: config.js [hunk 2]
```diff
@@ -70,5 +70,6 @@ module.exports = {
   PRETTY_DATE_FORMAT: 'Do MMMM YYYY',
   dateTimeFormat: 'DD MMM YYYY HH:mm:ss',
   dobFormat: 'YYYY-MM-DD',
-  dobCutoff: '2003-08-23'
+  // dbCutoff is the earliest date of birth that qualifies as under 18, therefore 27 August 2003 is considered over 18
```
Comment (jamiecarterHO) [inline config.js:73]: not hugely important but typo: `dbCutoff` -> `dobCutoff`

**Review by jamiecarterHO — APPROVED** [review]
Just a tiny typo, but approved

File: config.js (hunk 2 above)
Comment (michaelhartddat) [reply config.js]: EagleEye +1

**Pull request merged**